couch/check.py

```diff
@@ -119,7 +120,7 @@ def get_data(self, server, instance):
             databases = list(databases)[:self.MAX_DB]
 
         for dbName in databases:
-            url = urljoin(server, dbName)
+            url = urljoin(server, quote(dbName, safe = ''))
```

Comment:
Wondering if we might actually want to do this as:
```
url = urljoin(server, quote(dbName.encode('utf-8'), safe = ''))
```
so we can also cover edge unicode-related cases? Any thoughts on that?